src/IIIFPresentation/DLCS.Tests/API/DlcsApiClientTests.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System.Net;

```

Comment:
this file is CRLF - is that correct, or do we want LF?

Reply:
See https://github.com/dlcs/iiif-presentation/pull/200